MediaBrowser.Server.Mono/Program.cs

```diff
@@ -87,12 +88,26 @@ private static ServerApplicationPaths CreateApplicationPaths(string applicationP
         {
             if (string.IsNullOrEmpty(programDataPath))
             {
-                programDataPath = ApplicationPathHelper.GetProgramDataPath(applicationPath);
+                if (InteropServices.RuntimeInformation.IsOSPlatform(InteropServices.OSPlatform.Windows))
+                {
+                    programDataPath = Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData);
+                }
+                else
+                {
+                    // $XDG_DATA_HOME defines the base directory relative to which user specific data files should be stored.
+                    programDataPath = Environment.GetEnvironmentVariable("XDG_DATA_HOME");
+                    // If $XDG_DATA_HOME is either not set or empty, $HOME/.local/share should be used.
+                    if (string.IsNullOrEmpty(programDataPath)){
+                        programDataPath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".local", "share");
+                    }
+                }
             }
 
+            programDataPath = Path.Combine(programDataPath, "jellyfin");
```

Comment:
Can we add this as a conditional if the programDataPath doesn't already contain the string `jellyfin`? This results in duplicate directory structures in the Debian build:

```
$ ps aux | grep jellyfin
jellyfin  3676  0.8  4.9 3488040 100916 ?      SLsl 16:36   0:06 /usr/lib/jellyfin/bin/jellyfin -programdata /var/lib/jellyfin -restartpath /usr/lib/jellyfin/restart.sh
```
```
$ ls /var/lib/jellyfin/jellyfin/
cache  config  data  ffmpeg  localization  logs  root
```

Reply:
>basically, if the user wants to be explicit we should let them and not create a second level subdir

Fixed in #159.